### ISSUE TITLE
Error on setup for date format

### ISSUE THREAD

**Issue by vadimnazaroff** [GuyKh/ims-envista-custom-component#13]
### System Health details

V0.0.8

### Checklist

- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of any [previous issues](https://github.com/ludeeus/integration_blueprint/issues?q=is%3Aissue+label%3A%22Bug%22+)..

### Describe the issue

Error on setup for date format

### Reproduction steps

Setup integration
Add token
Select ALL data to load
Error on integration setup appears 

### Debug logs

```text
Logger: custom_components.ims_envista
Source: helpers/update_coordinator.py:354
integration: IMS Envista (documentation, issues)
First occurred: 10:37:22 (4 occurrences)
Last logged: 10:38:00

Unexpected error fetching ims_envista data
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 354, in _async_refresh
    self.data = await self._async_update_data()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/ims_envista/coordinator.py", line 101, in _async_update_data
    await self.config_entry.runtime_data.client.get_daily_station_data(
  File "/usr/local/lib/python3.12/site-packages/ims_envista/ims_envista.py", line 190, in get_daily_station_data
    return station_meteo_data_from_json(await get(session=self._session, url=get_url, token=self._token))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/ims_envista/meteo_data.py", line 247, in station_meteo_data_from_json
    meteo_data = [meteo_data_from_json(station_id, single_meteo_data) for single_meteo_data in data]
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/ims_envista/meteo_data.py", line 203, in meteo_data_from_json
    t = time.strptime(str(int(time_val)), "%H%M")
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/_strptime.py", line 548, in _strptime_time
    tt = _strptime(data_string, format)[0]
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/_strptime.py", line 333, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '7' does not match format '%H%M'
```

**Comment by GuyKh** [comment]
@vadimnazaroff - can you enable debug logs and printout the response from IMS? (response of `/daily` call)
and please share your station

**Comment by vadimnazaroff** [comment]
Sure, PFB:

2024-10-29 11:35:41.540 DEBUG (MainThread) [ims_envista.commons] Sending GET from https://api.ims.gov.il/v1/envista/stations/186/data//latest
2024-10-29 11:35:41.603 DEBUG (MainThread) [custom_components.ims_envista] Station 186 latest data: Station: 186, Date: **2024-10-29 13:20:00+02:00, Readings: [(TD: 29.4°C), (TDmax: 29.6°C), (TDmin: 29.1°C), (TG: 38.1°C), (RH: 23.0%), (Rain: 0.0mm), (WS: 2.6m/s), (WSmax: 6.0m/s), (WD: 319.0deg), (WDmax: 314.0deg),  (STDwd: 27.8deg), (WS1mm: 4.2m/s), (WS10mm: 2.6m/s), (Time: 12:20:00hhmm)]**

2024-10-29 11:35:41.604 DEBUG (MainThread) [ims_envista.commons] Sending GET from https://api.ims.gov.il/v1/envista/stations/186/data//daily
2024-10-29 11:35:42.037 ERROR (MainThread) [custom_components.ims_envista] Unexpected error fetching ims_envista data
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 354, in _async_refresh
    self.data = await self._async_update_data()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/ims_envista/coordinator.py", line 101, in _async_update_data
    await self.config_entry.runtime_data.client.get_daily_station_data(
  File "/usr/local/lib/python3.12/site-packages/ims_envista/ims_envista.py", line 190, in get_daily_station_data
    return station_meteo_data_from_json(await get(session=self._session, url=get_url, token=self._token))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/ims_envista/meteo_data.py", line 247, in station_meteo_data_from_json
    meteo_data = [meteo_data_from_json(station_id, single_meteo_data) for single_meteo_data in data]
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/ims_envista/meteo_data.py", line 203, in meteo_data_from_json
    t = time.strptime(str(int(time_val)), "%H%M")
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/_strptime.py", line 548, in _strptime_time
    tt = _strptime(data_string, format)[0]
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/_strptime.py", line 333, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '7' does not match format '%H%M'

**Comment by vadimnazaroff** [comment]
Station 24.Newe Yaar

**Comment by GuyKh** [comment]
https://github.com/GuyKh/py-ims-envista/pull/80

**Comment by GuyKh** [comment]
Should be fixed in 0.0.9

**Comment by vadimnazaroff** [comment]
Works on 0.0.9
